packages/web-component/test/descope-wc.test.ts

```diff
@@ -143,13 +144,22 @@ describe('web-component', () => {
       }
     });
     (createSdk as jest.Mock).mockReturnValue(sdk);
+
+    const origAppend = document.body.append;
+    const spyAppend = jest.spyOn(document.body, 'append');
+    spyAppend.mockImplementation((ele: any) => {
+      setTimeout(() => {
+        ele.onload?.();
+      });
+      origAppend.bind(document.body)(ele);
+    });
```

Comment:
you can extract this block to a helper function

Reply:
Done